Sources/Component/Action/ActionViewModel.swift

```diff
@@ -15,14 +29,20 @@ public struct ActionViewModel<Content: ViewModelProtocol>: ViewModelProtocol {
     /// The default initializer of `ActionViewModel`.
     ///
     /// - Parameter content: A generic state view model for the embedded `Content` view.
+    /// - Parameter highlightAnimation: The highlight animation to perform.
+    /// - Parameter cornerRadius: Set this to be able to have a rounded action view
     /// - Parameter animationDuration: The duration of the tab gesture highlighting animation.
     /// - Parameter action: The action closure to be called when a tab gesture is recognized
     public init(
         content: Content = ActionViewModel.default.content,
+        highlightAnimation: HighlightAnimation = ActionViewModel.default.highlightAnimation,
+        cornerRadius: CGFloat? = ActionViewModel.default.cornerRadius,
         animationDuration: TimeInterval = ActionViewModel.default.animationDuration,
```

Comment:
`cornerRadius` and `animationDuration` is just necessary for the highlighting, right? Maybe we can wrap it together in some kind of HighlightingConfiguration? What do you think?

Reply:
The `cornerRadius` is needed so the button can also be round, otherwise if the contentView has rounded corners the button is still rectangular. 

The `animationDuration` could be wrapped together with the `highlightAnimation`. What do you think about that?

Reply:
Yep sounds good! I would wrap it.

Reply:
Done.